### PR TITLE
readme: Get back in sync for #6 and #17

### PR DESCRIPTION
**Pull request by hartwork**
Related to #6 and #17

What do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,4 +134,5 @@ google is up
 
 ## License
 
-MIT &copy; [**Travis Clarke**](https://blog.travismclarke.com/)
+MIT &copy; [**Travis Clarke**](https://blog.travismclarke.com/),
+           [Sebastian Pipping](https://blog.hartwork.org/)
```
Comment (clarketm) [inline README.md:138]: Feel free to add your name to the [LICENSE](https://github.com/clarketm/wait-for-it/blob/master/LICENSE) as well! Also, I sent you a collaborator invite.

**Comment by clarketm** [comment]
Thanks for also updating the docs as well! After these changes go in, we will be ready to cut a new minor release!

File: README.md (hunk 1 above)
Comment (hartwork) [reply README.md:138]: Very kind, thank you! :pray: I have added myself to `LICENSE` now and I gladly accept the invitation.

**Comment by hartwork** [comment]
> The changes look good, but can you please also add these updates to the docs [index.md](https://github.com/clarketm/wait-for-it/blob/master/docs/source/index.md). Currently, (for formatting reasons) I have these as separate files.

Sure, done.

> Thanks for updating the docs as well! After these changes go in, we will be ready to cut a new minor release.

That would be awesome.

**Pull request merged**